pkg/tenant/members.go

```diff
@@ -35,6 +84,57 @@ func (s *Server) MemberList(c *gin.Context) {
 	c.JSON(http.StatusNotImplemented, "not implemented yet")
 }
 
+// MemberCreate adds a new member to the database and returns
+// a 201 StatusCreated response.
+//
+// Route: /member
+func (s *Server) MemberCreate(c *gin.Context) {
+	var (
+		err    error
+		member *api.Member
+	)
+
+	// Bind the user request and return a 400 response if binding
+	// is not successful.
+	if err = c.BindJSON(&member); err != nil {
+		log.Warn().Err(err).Msg("could not bind member create request")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not bind request"))
+		return
+	}
+
+	// Verify that a member id does not exist and return a 400 response if
+	// the member id exists.
+	if member.ID != "" {
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("member is cannot be specified on create"))
+		return
+	}
```

Comment:
In this case, we will want to assert that the `TenantID` is on the member struct and is parseable as a ULID right?

Reply:
There isn't a TenantID associated with the  api.Member struct, only an id for the member.  There's a TenantID on the TenantMemberPage struct that'll be used with TenantMemberList. This is why I was wondering if there should be separate member models in the db.

Reply:
Ok, let's pause on this and follow up on it later - it might be Quarterdeck related. Go ahead and merge this PR.